### ISSUE TITLE
[req]: OKPT

### ISSUE THREAD

**Issue by garfield69** [Jackett/Jackett#14534]
### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Is there already a request for your tracker?

- [X] I have checked older issues, open and closed

### Type

Private

### Tracker details

```markdown
- Name: OKPT
- URL: https://www.okpt.net/
- Description: OKPT is a CHINESE Private site for MOVIES / TV / MUSIC / GENERAL
- Minimum Seeding Time and Minimum Ratio (if Private): tbd
- Additional Comments: 
Engine: NexusPHP v1.8.3 2023-05-14
Registration open ATM
Exam: UL 30GB, DL 15GB, Bonus 500
```

**Comment by garfield69** [comment]
v0.21.372

**Comment by garfield69** [comment]
https://www.okpt.net/ cloudflare Connection timed out Error code 522

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://www.okpt.net/ HTTP ERROR 500 (Internal Server Error)

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14534#issuecomment-2192391816

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
[#14534 (comment)](https://github.com/Jackett/Jackett/issues/14534#issuecomment-2192391816)

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14534#issuecomment-2192391816

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14534#issuecomment-2192391816

**Comment by garfield69** [comment]
online

**Comment by ilike2burnthing** [comment]
https://www.okpt.net/ - A timeout occurred Error code 524

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://www.okpt.net/ > cloudflare Origin is unreachable Error code 523

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://www.okpt.net/ > Sorry! This site has been shut down by the administrator, please contact the administrator for details!

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
https://www.okpt.net/ > HTTP ERROR 500

**Comment by garfield69** [comment]
online

**Comment by ilike2burnthing** [comment]
https://www.okpt.net/ - The website is currently inaccessible...

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
why can I not access okpt from firefox

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/4f4240e7-51c0-4286-8a42-bb9210844566" />

But I can from chrome and Jackett?

Just me?

**Comment by ilike2burnthing** [comment]
Jackett working ok for me, but browsers aren't. Using my saved passkey login link just returns 404. Maybe cookie, but I'm not sure.

**Comment by garfield69** [comment]
I copied my cookies over from chrome (which also match the ones in my jackett okpt.json from the indexers folder), across to my firefox session, and refreshed, but that did not work :-(
Never mind, I'll just use firefox for the rest and chrome for okpt for now.
As for chrome, I did get a http 500 when using the domain on its own, but got a page when I added /index.php, so go figure.

**Comment by garfield69** [comment]
yeah, the saved passkey also gave me 404 and the user-cp pages do not have an option to generate a login passkey link anymore
I'll browser the forum to see if there is any info
[late edit] nothing found in forum but an outdated entry back in 2023 that said the option was available
clearly the site lost the feature when they upgraded nexus

**Comment by ilike2burnthing** [comment]
https://github.com/Jackett/Jackett/issues/14534#issuecomment-2985124308

Failing in Jackett now too.

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14534#issuecomment-2985124308

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14534#issuecomment-2985124308

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
- https://www.okpt.net/ > cloudflare  Bad gateway Error code 502 
- Error code: 500 Internal Server Error

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://www.okpt.net/ > 403 Forbidden nginx

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://www.okpt.net/ > cloudflare Web server is down Error code 521

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/14534#issuecomment-3980427440

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://www.okpt.net/ > Error 500 Internal Server Error

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
- https://www.okpt.net/ > cloudflare Connection timed out Error code 522
- 500 Internal Server Error

**Comment by garfield69** [comment]
online
